app/src/main/java/io/intrepid/contest/screens/contestcreation/editcategoriestocontest/EditCategoriesFragment.java

```diff
@@ -80,9 +87,20 @@ public void showCategoriesList() {
         activity.showCategoryList();
     }
 
+    @Override
+    public void showCategory(String previousCategoryName, String previousCategoryDescription) {
```

Comment:
I see! When I read it I thought it was "previous category" name, not previous "category name". You can consider it the current category name, it's not really previous (you might not change it)